### DISCUSSION TITLE
Booting with a non-integrated network adapter

### DISCUSSION THREAD

**Discussion by Mogaba** [ltsp/ltsp#815] [Q&A] [answered]
I have a PC with a faulty integrated network adapter. I installed a PCI-E TP-Link adapter but I can't boot with it: BIOS doesn't show it as a boot option. I think it doesn't have a chip for booting (BootROM?)

Here's the specification for the adapter:
https://www.tp-link.com/en/home-networking/pci-adapter/tg-3468/#specifications

Are there any workarounds?

Also, are there any tips on how to know if an adapter supports booting just by looking at it? Or you always need to check specs?

**Comment by alkisg** [comment, marked as answer]
See the docs: http://ltsp.org/docs/netboot-clients/#ipxe
